engine/restful_router.go

```diff
@@ -93,7 +93,7 @@ func newRouter(isREST bool) *mux.Router {
 				"HTTP Go performance profiler (pprof) endpoint enabled: http://%s:%d/debug/pprof\n",
 				common.ExtractHost(listenAddr),
 				common.ExtractPort(listenAddr))
-			router.PathPrefix("/debug").Handler(http.DefaultServeMux)
+			router.PathPrefix("/debug/pprof/").HandlerFunc(pprof.Index)
```

Comment:
Need to add a slash "/" at the end and have the profile enabled, will update the log comment to reflect that

Reply:
Knew I forgot to test something! pprof enabled in config. run gct, then do:
` go tool pprof -http "localhost:8085" 'http://localhost:9050/debug/pprof'` gets 404
ie me too

Reply:
adding slash works

Reply:
if you remove the / from 

```
router.PathPrefix("/debug/pprof/").HandlerFunc(pprof.Index)
```

the route will be detected properly for both 

```
gocryptotrader on  linter_updates [!?] via 🐹 v1.14 
✗ curl -I "localhost:9050/debug/pprof"
HTTP/1.1 200 OK
Date: Thu, 09 Apr 2020 01:13:32 GMT
Content-Type: text/html; charset=utf-8


gocryptotrader on  linter_updates [!?] via 🐹 v1.14 
➜ curl -I "localhost:9050/debug/pprof/" 
HTTP/1.1 200 OK
Date: Thu, 09 Apr 2020 01:13:35 GMT
Content-Type: text/html; charset=utf-8
```

(fun fact firefox appears to support this by default or it was cached since i didn't have this during testing)